Sources/PointFree/Auth.swift

```diff
@@ -86,8 +86,8 @@ public func currentSubscriptionMiddleware<A, I>(
     )
     ?? throwE(unit)
 
-  return (userSubscription.run.parallel <|> ownerSubscription.run.parallel)
-    .sequential
+  return (userSubscription <|> ownerSubscription)
+    .run
```

Comment:
this was causing a race condition in my tests. not sure why I raced the parallels instead of using the first success of `EitherIO`.

Reply:
maybe we need `EitherPar<E, A>`